app/templates/components/events/view/export/download-zip.hbs

```diff
@@ -10,23 +10,28 @@
 <div class="ui container"></div>
 <div class="ui form">
   <div class="field">
-    {{ui-checkbox class='toggle' label=(t 'Image')}}
+    {{ui-checkbox class='toggle' label=(t 'Image') checked=dataImage onChange=(action (mut dataImage))}}
```

Comment:
@niranjan94 Should I include the dataImage, dataVideo, etc properties in controller? Since we are not keeping the model layer. Will that be okay?